generators/app/templates/app/scripts/app.js

```diff
@@ -14,7 +14,7 @@ const sblocksComponents = [
   // simple-blocks implementation
   // https://github.com/lexich/simple-blocks
 ];
-const Modernizr = window.Modernizr;
+// const Modernizr = window.Modernizr;
```

Comment:
Modernizr is available globally. This line means nothing. Write usual comment to inform developer.